packages/interface-ipfs-core/src/files/write.js

```diff
@@ -174,21 +169,17 @@ module.exports = (common, options) => {
     })
 
     it('writes a small file using a Node stream (Node only)', async function () {
-      if (!isNode) {
-        return this.skip()
-      }
-
+      if (!isNode) this.skip()
```

Comment:
Was changing this to a one-line if necessary?  I can't help but think of [this bug](https://www.imperialviolet.org/2014/02/22/applebug.html) every time I see a one-line if statement.

Reply:
ofc not... changing it back